purchases_ui_flutter/lib/views/paywall_footer_view.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:purchases_flutter/models/offering_wrapper.dart';
+
+import 'internal_paywall_footer_view.dart';
+
+/// View that displays the paywall in footer mode.
+/// Not supported in macOS currently.
+///
+/// [offering] (Optional) The offering object to be displayed in the paywall.
+/// Obtained from [Purchases.getOfferings].
+///
+/// [content] The content to be displayed above the paywall. Make sure you
+/// apply a padding to the bottom of your content to avoid overlap. This padding
+/// can be obtained from [PaywallFooterView.roundedCornerRadius].
+class PaywallFooterView extends StatefulWidget {
+  /// The top corner radius of the footer view.
+  static const roundedCornerRadius = 20.0;
+
+  final Offering? offering;
+  final Widget content;
+
+  const PaywallFooterView({
+    Key? key,
+    this.offering,
+    required this.content,
+  }) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _PaywallFooterViewState();
+}
+
+class _PaywallFooterViewState extends State<PaywallFooterView> {
+
+  // Need to set it to a value > 0 so it's drawn. Setting it to a value that
+  // approximately reflects what the footer view height will be, so redrawing
+  // is not so noticeable. Need to improve this.
+  var _height = 264.0;
```

Comment:
Doesn't this depend a lot on the number of packages for example?

Reply:
Yes, this is just an approximation, so it doesn't start with a non-visible footer and then it suddenly appears. This wouldn't be a problem without the delay I mentioned before when drawing using `PlatformView` in Android.